CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 What's New on LIAC-ARFF
 =======================
 
+LIAC-ARFF 2.3.2
+* fix: match all possible separator spaces to add quotes to be compliant with
+  ARFF format when writing.
```

Comment:
You've said "writing" here but "The decoding is however the place which is going wrong" in your comment.

Reply:
Oh my comment was really misleading. What I meant about this comment was the following.
The encoding of the `\r\n` and `\n` into ARFF file is well handled (with quote around the string). The issue with `\r\n` and `\n` is happening during the ARFF decoding which as we discuss in #89

Reply:
Could you please fix the formatting of your what's new message?